test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTests.cs

```diff
@@ -27,22 +27,19 @@ public SqlClientIntegrationTests(SqlClientIntegrationTestsFixture fixture)
     [InlineData(CommandType.Text, "select 1/1")]
     [InlineData(CommandType.Text, "select 1/1", true, "select ?/?")]
     [InlineData(CommandType.Text, "select 1/0", false, null, true)]
-    [InlineData(CommandType.Text, "select 1/0", false, null, true, false, false)]
-    [InlineData(CommandType.Text, "select 1/0", false, null, true, true, false)]
+    [InlineData(CommandType.Text, "select 1/0", false, null, true, true)]
```

Comment:
Do you get the same failure locally as I get as fixed in #2826?

Reply:
No, these tests were running fine for me on a Windows 11 machine. A test case was removed here because I removed the `shouldEnrich` parameter. I do not feel that it is important that this integration test tests enrich. We have that coverage elsewhere.